day_7/fizzbuzz.js

```diff
@@ -0,0 +1,11 @@
+for (var i = 0; i <= 100; i++) {
+  if ( i % 3 === 0 && i % 5 === 0) {
```

Comment:
nice work here! I would check mdn to see how if else statements should be spaced!